browser/profiles/brave_profile_manager.cc

```diff
@@ -44,6 +44,7 @@
 #endif
 
 #if BUILDFLAG(ENABLE_IPFS)
```

Comment:
why is a brave wallet service guarded by enabled_ipfs, they are not guarded by enable_ipfs in the BUILD.gn

Reply:
header buildflags and gn buildflags should always match regardless of whether the code is only use when some other buildflag is enabled